WalletWasabi/WabiSabi/Backend/Banning/CoinVerifier.cs

```diff
@@ -240,11 +246,11 @@ public bool TryScheduleVerification(Coin coin, [NotNullWhen(true)] out CoinVerif
 						Whitelist.Add(coin.Outpoint);
 					}
 
-					item.SetResult(new CoinVerifyResult(coin, ShouldBan: shouldBan, ShouldRemove: shouldBan));
+					item.SetResult(new CoinVerifyResult(coin, ShouldBan: shouldBan, ShouldRemove: shouldBan, Reason.RemoteApiChecked, apiResponseItem));
 				}
 				catch (Exception ex)
 				{
-					item.SetResult(new CoinVerifyResult(coin, ShouldBan: false, ShouldRemove: true));
+					item.SetResult(new CoinVerifyResult(coin, ShouldBan: false, ShouldRemove: true, Reason.Exception));
```

Comment:
The exception message is not added to the details field?

Reply:
Not yet